docs/ja/user-guide/commands.md

```diff
@@ -72,6 +72,14 @@ name: bar # コマンドの名前
 version: '1.0' # メジャーバージョンとマイナーバージョン (パッチバージョンは自動付与)
 description: |
   Lorem ipsum dolor sit amet.
+usage: |
+  sd-cmd exec foo/bar@  
```

Comment:
This line should be ` sd-cmd exec foo/bar@<VERSION> <OPTION> <TARGET>`

Reply:
We should use `[]`, `...` and so on.
e.g. `cp [OPTION]... SOURCE... DIRECTORY`